demos/CMakeLists.txt

```diff
@@ -6,6 +6,8 @@ cmake_minimum_required (VERSION 2.8.12)
 
 project(Demos)
 
+option(WITH_PYTHON "Whether to build extension modules for Python demos" OFF)
```

Comment:
I suggest `ENABLE_PYTHON` to stay consistent with IE.

Reply:
OK, done.